package.json

```diff
@@ -1,48 +1,54 @@
 {
-  "name": "@signalk/freeboard-sk",
-  "version": "0.0.4",
-  "description": "Openlayers chartplotter implementation for Signal K",
-  "main": "index.js",
-  "dependencies": {},
-  "browser": {
-    "bootstrap-drawer": "./node_modules/bootstrap-drawer/dist/js/drawer.js",
-    "bootstrap-slider": "./node_modules/bootstrap-slider/js/bootstrap-slider.js",
-    "bootstrap-toggle": "./node_modules/bootstrap-toggle/js/bootstrap-toggle.js",
-    "bootstrap-select": "./node_modules/bootstrap-select/js/bootstrap-select.js"
-  },
-  "devDependencies": {
-    "@signalk/client": "^0.1.4",
-    "bluebird": "^2.9.24",
-    "bootstrap": "^3.3.4",
-    "bootstrap-drawer": "^1.0.0",
-    "bootstrap-select": "^1.6.3",
-    "bootstrap-slider": "^4.8.0",
-    "bootstrap-toggle": "^2.0.0",
-    "browser": "^0.2.6",
-    "browserify": "^9.0.8",
-    "debug": "^2.1.3",
-    "font-awesome": "^4.6.3",
-    "jquery": "^2.1.3",
-    "karma": "^0.13.15",
-    "knob": "^0.3.0",
-    "lodash": "^4.17.4",
-    "mocha": "^2.3.4",
-    "nanomodal": "^5.1.0",
-    "openlayers": "^3.16.0",
-    "simplify-js": "^1.2.1",
-    "uglify-js": "^2.4.20",
-    "watchify": "^3.1.2",
-    "web-audio-daw": "^2.3.1",
-    "xml2js": "^0.4.16"
-  },
+  "name": "freeboard",
```

Comment:
@signalk

Reply:
Folder restructured to move angular project source to the `source` folder and the deployable files are located in the `root` .npmignore added to exclude `/source`

Reply:
Um, where do we continue this work? What repo, what branch?

Reply:
I think we have evolved beyond a port of an application to more of an inter-operability exercise that is surfacing issues around areas of the spec / server implementations where there are still questions to be answered.

Most of the discussion has been around how the different server implementations differ:
- `/resources/charts` response
- anchor alarm operation
- how to list what applications are installed
The things that an application needs to know to be able to present a "better" user experience.

So ultimately what I think we are getting into here is `what does a well structured /well  behaved Signal K application look like and what does it need from the server?`
- How does it test the server for available capabilities? i.e. history api
- How does it query what applications / plugins are available?
- Is security enabled?
- If it can't PUT values to API paths that needs what should it do? Fail with a message, try an alternative (e.g. send it via an UPDATE message), etc.
- and more......

Happy to take this to a new repo if it makes sense and maybe position the application as a means for testing server api / spec implementations to ensure an application can work across all server types.

Prior to the port there was Freeboard functionality that did not work on node server but mostl likely did *(I have only used the node server so I am making an assumption here)* on the Java / Artemis server. We are now testing the port for similar operation on both server types which is where most of the dialog is.

Reply:
As things did not work perfectly before I don't think we need to resolve all problems before publishing. Your work is a delight and a major step forward I'd like to push this out and take care of the smaller gripes piecemeal afterwards.

One major advantage in publishing this as a new version of Freeboard is that people who have Freeboard already installed will get notified of the available upgrade and can upgrade with one click.

I'll continue the discussion in #51.